### PR TITLE
The curious case of Ms certs

### PR DESCRIPTION
**Pull request by MrSeccubus**

### CODE REVIEW

**Comment by MrSeccubus** [comment]
ERROR CasesPlugin: Case DIVD-2022-00013 does not have an excerpt or the default excerpt

File: _cases/2022/DIVD-2022-00013.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+layout: case
+title: The curious case of the odd update.microsoft.com certificates
+excerpt: "Please set this to and except of a few lines.
```
Comment (Lennaert89) [inline _cases/2022/DIVD-2022-00013.md:4]: No Excerpt?

File: _cases/2022/DIVD-2022-00013.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+timeline:
+- start: 2022-02-05
+  end:
+  event: "Certificates discovered for first time"
+- start: 2022-08-24
+  end:
+  event: "Case is referred to the ethics committee to see if it fits into the CoC"
+- start: 2022-09-19
+  end:
+  event: "Ethics committee, ruels that case is within CoC"
```
Comment (Lennaert89) [inline _cases/2022/DIVD-2022-00013.md:32]: typo

**Review by Lennaert89 — CHANGES_REQUESTED** [review]

**Comment by MrSeccubus** [comment]
See #499

**Pull request closed without merging**